component_documentation/componentDoc.css

```diff
@@ -65,7 +65,7 @@ body {
     position: fixed;
     left: 0;
     width: 20rem;
-    height: 100vh;
+    height: 100vh-67px;
```

Comment:
You can use calc() for such calculations

Reply:
thank you, i will do that